_config/2fa.yml

```diff
@@ -15,3 +15,11 @@ Injector:
     class: '_2fa\CMSProfileController'
 CMSSecurity:
   reauth_enabled: false
+
+LeftAndMain:
+  extra_requirements_css:
+    - silverstripe-twofactorauth/css/twofactorstyles.css
```

Comment:
"installer-name": "twofactorauth", in composer.json so the path here should be `twofactorauth` and avoid duplicate.